src/main/java/common/DriverBinariesSetter.java

```diff
@@ -1,40 +1,134 @@
 package common;
 
+import com.codeborne.selenide.WebDriverRunner;
+import config.ConfigurationManager;
 import io.github.bonigarcia.wdm.WebDriverManager;
 import org.apache.commons.lang3.StringUtils;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.edge.EdgeOptions;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.firefox.FirefoxOptions;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import static common.DriverBinariesSetter.Drivers.*;
+
 
 public final class DriverBinariesSetter {
```

Comment:
Really class brakes SRP. It's responsible for choosing a browser and fine setting especially after adding classes callSauceLabProperties